content/support.rst

```diff
@@ -0,0 +1,212 @@
+Support
+#######
+
+:menu_order: 009
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describes Support details.
+
+For access to any of the Support platforms, please contact the Support team.
+
+For the full details on the Support offering, please go to the website at 
+
+
+Helpdesk
+========
+
+Each message to the Support email starts a helpdesk ticket in Freshdesk.
+
+The helpdesk software is currently under evaluation.
+Currently the main point of contact for Support and liaising with customers
+is via the Google Group email.
+
+
+Template: Obtaining the installation detail
+-------------------------------------------
+
+Can you please provide us with the following details about the system used for running the SFTPPlus Server or Client:
+
+.. sourcecode:: rst
+
+	 * SFTPPlus Server or Client?
+
+	 * SFTPPlus product version?
+
+	 * If Server – do you also use WebAdmin?
+
+	 * Operating system name?
+
+	 * Operating system version?
+
+	 * CPU architecture?
+
+	 * File transfer protocol(s) used?
+
+	 * Is this installation part of a resilience/redundancy/high availability architecture?
+
+	 * Other information (if known). Please advise brief details of any
+	 integration, software, security or other technologies that interact
+	 with SFTPPlus products. e.g. Firewalls, Proxy, LDAP, Active Directory,
+	 Single Sign On, Management software etc
+
+	Before releasing a new version of SFTPPlus products or any fix, we run
+	an automated test suite to validate new features and check for
+	regressions. By knowing a few details about your installations we can
+	configure the test suite to run against a setup similar to your
+	installation. This information will not be shared with any other third
+	party and will be used solely for improving further support for you and
+	development of the product.
+
+
+Using the installation details
+------------------------------
+
+Once the installation details are provided, recreate the issue in a similar
+environment.
+
+Access the buildslave page of the OS to obtain the IP address and SSH into
+the IP.
+
+Download the initialize the SFTPPlus version that is the same as the customer's
+and initialise.  Do not test with the buildslave version.
+
+
+Knowledge Base
+==============
+
+Currently the main knowledge base is in documentation - both on the website
+and in the internal documentation pages supplied with each package.
+
+A separate knowledge base is currently under evaluation.
+
+
+Chat
+====
+
+`Olark <https://www.olark.com>`_ is currently used for chat.
+
+Each chat transcript is forwarded to the Support email list and subsequently
+creates a new ticket in the helpdesk.
```

Comment:
> Also, for terminology, maybe we can refer to a support ticket as "support case" so than when we say support case we know that is something in our suppport system... while a support ticket can be someting inside our development system

I am changing it to mention if a ticket is a `helpdesk ticket` or if it's a `Trac ticket`.
I don't want to mention `Freshdesk ticket` since that is still under evaluation.